src/components/DataTable/TableBatchEditable.js

```diff
@@ -0,0 +1,37 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import classnames from 'classnames';
+import Table from './Table';
+
+/**
+ * `<Table>` supporting batch editing UI.
+ */
+const TableBatchEditable = ({ children, className, saving, ...other }) => {
```

Comment:
Would this read better as `BatchEditableTable` potentially?

Reply:
OK made a change as such!